content/en/blog/_posts/2019-12-10-Gamified-Chaos-Engineering-Tool-for-Kubernetes.md

```diff
@@ -0,0 +1,120 @@
+**KubeInvaders - Gamified Chaos Engineering Tool for Kubernetes**
```

Comment:
Remove bolded title and add front matter: 

---
layout: blog
title: "Gamified Chaos Engineering Tool for Kubernetes"
date: 2019-12-18
---

**Author:** Eugenio Marzo, SourceSense